dockerHashing/src/main/scala/cromwell/docker/registryv2/DockerRegistryV2Abstract.scala

```diff
@@ -70,8 +70,15 @@ object DockerRegistryV2Abstract {
   }
 
   // Placeholder exceptions that can be carried through IO before being converted to a DockerInfoFailedResponse
-  private class Unauthorized() extends Exception
-  private class NotFound() extends Exception
+  private class Unauthorized(message: String, responseBody: String) extends Exception {
+    override def getMessage(): String = message + " " + responseBody
+  }
+  private class NotFound(message: String, responseBody: String) extends Exception {
+    override def getMessage(): String = message + " " + responseBody
+  }
+  private class UnknownError(message: String, responseBody: String) extends Exception {
+    override def getMessage(): String = message + " " + responseBody
```

Comment:
So unfortunately, the override seems necessary to pipe the new arg to the message method. I did combine the inputs, and I do wonder if there is a better way to extend java exceptions, but I think this will have to do for now.

Reply:
I think you might need:
```
private class Unauthorized(val message: String) extends Exception(message)
```
...but I also think it's fine as is for now.